kube/services/revproxy/gen3.nginx.conf/arborist-service.conf

```diff
@@ -92,6 +92,25 @@ location = /authz/mapping {
     proxy_pass $upstream/auth/mapping;
 }
 
+location = /authz/policy {
+    if ($csrf_check !~ ^ok-\S.+$) {
+        return 403 "failed csrf check";
+    }
+
+    # authz check to see if the user has access to the policy data
+    set $authz_resource "/services/arborist/policy";
+    set $authz_method "read";
+    set $authz_service "arborist";
+    # be careful - sub-request runs in same context as this request
+    auth_request /gen3-authz;
+
+    set $proxy_service ${arborist_release_name};
+    set $upstream http://${arborist_release_name}-service.$namespace.svc.cluster.local;
```

Comment:
Why the rewrite and proxy_redirect? Can you not just proxy_pass?

Reply:
we need to pass the query parameters too